frontend/src/routes/Login.tsx

```diff
@@ -75,6 +77,13 @@ const Login: React.FC<Props> = ({ queryRef }) => {
                 display: "flex",
                 flexDirection: "column",
             }}>
+                <div css={{
+                    position: "absolute",
+                    left: 16,
+                    margin: OUTER_CONTAINER_MARGIN,
+                }}>
+                    <Breadcrumbs path={[]} tail={t("login-page.heading")} />
+                </div>
```

Comment:
Mh I'm not terribly thrilled by the `position: "absolute"` solution. For one, the `margin` doesn't do anything here, so that should be removed in this case. What about this alternative:

- Move most of the CSS of `<main>` into a new div thats child of `<main>`, and put all old elements (header, box, ..) into that div. The `padding: 16` can be removed then I think.
- Add the `margin: OUTER_CONTAINER_MARGIN` to `<main>`

I think that should work fine?

Reply:
Oh woops, my bad. Didn't test with a large enough screen, yes margin does something.

Re pushing down other stuff: in that case we should just reduce the `margin-top` of the h1.